### ISSUE TITLE
End of day screen shrink

### ISSUE THREAD

**Issue by G4M3RL1F3** [RecompRando/MMRecompRando#13]
Whenever a day is about to end, the screen shrinks. It would be a nice quality of life if this shrinking could be removed. 🙂

**Comment by LittleCube-hax** [comment]
This one might be rough. It'll take a little more research before I know exactly where to look.

**Comment by CelestialKitsune** [comment]
> This one might be rough. It'll take a little more research before I know exactly where to look.

For what it's worth, that's mannaged by the `En_Test4` actor in the `EnTest4_GetBellTimeAndShrinkScreenBeforeDay3` function that changes the `screenScale` field of the global `SaveContext` structure

**Comment by githubplanetarparrot** [comment]
Isn't this and intended function in the game?

**Comment by TSEG6** [comment]
I do actually like the screen shrink so I would hope it would be a toggle if implemented.

**Comment by ThatHypedPerson** [comment]
Was told this would be done in a separate QoL mod. Closing.

**Comment by CelestialKitsune** [comment]
> Was told this would be done in a separate QoL mod. Closing.

Yup! Here it is. https://thunderstore.io/c/zelda-64-recompiled/p/CelestialMods/Shrink_Skip/
